### ISSUE TITLE
Issues with editor content submission using keyboard shortcuts

### ISSUE THREAD

**Issue by AbhayVAshokan** [neetozone/neeto-editor#143]
- [x] Add ability to clear editor content on submission.
- [ ] `onSubmit` is returning the initial value instead of the updated value: [codesandbox by vivekpatel-77](https://codesandbox.io/s/inspiring-rosalind-ctc2l).

Ref: 
  1. https://github.com/bigbinary/neeto-planner-web/pull/536#discussion_r782206049
  2. https://github.com/bigbinary/neeto-planner-web/pull/542#discussion_r782751365

bug _t
@AbhayVAshokan _a

**Comment by rishavkr08** [comment]
@AbhayVAshokan what's the update here?

**Comment by AbhayVAshokan** [comment]
@rishavkr08 the first issue mentioned above is solved in #148.
Regarding the second issue, I had a call with @amaldinesh7 regarding the same. We couldn't come up with a solution as of now. The `onSubmit` and `onChange` functions maintain an earlier reference throughout the editor's lifetime. 

Please use the value directly obtained from the `onSubmit` function props to perform the required operations.

```js
const [value, setValue] = useState("initial state");

const handleSubmit = (htmlContent) => {
  console.log(htmlContent); // latest reference
  console.log(value);       // old reference
}
```

**Comment by AbhayVAshokan** [comment]
Closing this issue. Since the controlled mode is removed from the editor, the content state cannot be maintained at the project level and hence will not be possible to encounter this issue.

The `onSubmit` prop provides the latest version of the `htmlContent` and it should be used for obtaining the editor content.
